VideoLocker/src/main/java/org/edx/mobile/player/TranscriptManager.java

```diff
@@ -182,58 +148,22 @@ public void downloadTranscriptsForVideo(TranscriptModel transcript){
         }
 
         if(transcript.chineseUrl!=null){
-            try {
-                startTranscriptDownload(transcript.chineseUrl);
-            } catch (NoSuchAlgorithmException e) {
-                logger.error(e);
-            } catch (UnsupportedEncodingException e) {
-                logger.error(e);
-            }
+            startTranscriptDownload(transcript.chineseUrl);
```

Comment:
There seems to be a lot of copy/paste code here. If it's not too much trouble can we move all if this checking/downloading for each language into one helper function? I don't mind if we manually call invoke each language, but it seems redundant to check each language for non null and then callStartTranscriptDownload on each one